### PR TITLE
EWPP-297: Prevent deletion of vocabulary if referenced by an association

### PR DESCRIPTION
**Pull request by brummbar**
Prevent deletion of vocabulary if referenced by an association

### CODE REVIEW

**Comment by brummbar** [comment]
API deletion will be handled later on.

File: src/OpenVocabularyAssociationStorage.php [hunk 1]
```diff
@@ -78,6 +78,22 @@ public function loadAssociationsByField(string $field_id): array {
     return $this->loadMultiple($results);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function loadAssociationsByVocabulary(string $vocabulary_id): array {
```
Comment (upchuk) [inline src/OpenVocabularyAssociationStorage.php:84]: You introduce this but how come you don't use it anywhere? I'd have imaged in OpenVocabularyDeleteForm::buildForm where you run the same query no?

**Review by upchuk — CHANGES_REQUESTED** [review]

File: src/OpenVocabularyAssociationStorage.php (hunk 1 above)
Comment (brummbar) [reply src/OpenVocabularyAssociationStorage.php:84]: Oh I didn't push that it seems, true

**Review by upchuk — APPROVED** [review]

**Review by 22Alexandra — DISMISSED** [review]

**Review by upchuk — DISMISSED** [review]
Merge when OPENEUROPA-3258 is in.

**Review by 22Alexandra — APPROVED** [review]

**Review by upchuk — APPROVED** [review]

**Pull request merged**